server/tserver/src/main/java/org/apache/accumulo/tserver/NativeMap.java

```diff
@@ -60,39 +60,44 @@
  * would be a mistake for long lived NativeMaps. Long lived objects are not garbage collected quickly, therefore a process could easily use too much memory.
  *
  */
-
 public class NativeMap implements Iterable<Map.Entry<Key,Value>> {
 
   private static final Logger log = LoggerFactory.getLogger(NativeMap.class);
   private static AtomicBoolean loadedNativeLibraries = new AtomicBoolean(false);
 
   // Load native library
   static {
-    // Check standard directories
-    List<File> directories = new ArrayList<>(Arrays.asList(new File[] {new File("/usr/lib64"), new File("/usr/lib")}));
-    // Check in ACCUMULO_HOME location, too
-    String accumuloHome = System.getenv("ACCUMULO_HOME");
-    if (accumuloHome != null) {
-      directories.add(new File(accumuloHome + "/lib/native"));
-      directories.add(new File(accumuloHome + "/lib/native/map")); // old location, just in case somebody puts it here
+    // Check in directories set by 'accumulo.native.lib.dirs' system property
```

Comment:
Small suggestion, for naming consistency: If we're going to support multiple directories, using path syntax and the path separator, we should call this field `accumulo.native.lib.path`